lib/membrane_h264_plugin/parser/nalu_parser.ex

```diff
@@ -5,19 +5,17 @@ defmodule Membrane.H264.Parser.NALuParser do
 
   As a result the parser outputs a stream of parsed NAL units.
   """
-  alias Membrane.H264.Parser.{NALu, NALuSplitter, NALuTypes, SchemeParser}
+  alias Membrane.H264.Parser.{NALu, NALuTypes, SchemeParser}
   alias Membrane.H264.Parser.SchemeParser.Schemes
 
   @typedoc """
   A structure holding the state of the NALu parser.
   """
   @opaque t :: %__MODULE__{
             scheme_parser_state: SchemeParser.t(),
-            has_seen_keyframe?: boolean(),
-            prev_pts: non_neg_integer() | nil,
-            prev_dts: non_neg_integer() | nil
+            has_seen_keyframe?: boolean()
           }
-  @enforce_keys [:scheme_parser_state, :has_seen_keyframe?, :prev_pts, :prev_dts]
+  @enforce_keys [:scheme_parser_state, :has_seen_keyframe?]
   defstruct @enforce_keys
```

Comment:
Instead of enforcing keys let's provide defaults (similarly to nalu splitter)

Reply:
done